internal/ir/infra.go

```diff
@@ -76,7 +76,8 @@ type HTTP3Settings struct {
 // HTTP1Settings provides HTTP/1 configuration on the listener.
 // +k8s:deepcopy-gen=true
 type HTTP1Settings struct {
```

Comment:
missed this earlier, this struct should be living in `xds.go`

Reply:
Moved to `xds.go` in #2517